mani_skill/utils/building/_mjcf_loader.py

```diff
@@ -247,8 +247,17 @@ def _build_geom(
             )
 
         geom_density = _parse_float(geom_attrib, "density", 1000.0)
-        physx_material = None
-        # TODO handle geometry material properties
+
+        # if condim is 1, we can easily model the material's friction
+        if _parse_int(geom_attrib, "condim", 0) == 1:
+            friction = _parse_float(
+                geom_attrib, "friction", 0.3
+            )  # maniskill default frection is 0.3
```

Comment:
frection -> friction

Reply:
merged, and everything works - learns as consistent as as quick as before